starter-code/styles/base.css

```diff
@@ -1,6 +1,7 @@
 /* TODO: Add a rule for images, so they never exceed their container's width. */
-
-
+img {
+    width: 100%;
```

Comment:
I think the intent here was to use "max-width", but this technically satisfies the requirement. ;)